checkQC/web_app.py

```diff
@@ -49,6 +51,28 @@ def get(self, runfolder):
                                                       "type of instrument/run configuration on the run you want to "
                                                       "analyze is available in the qc config?")
 
+    def prepare(self):
+        if self.request.body:
+            try:
+                self.args = json.loads(self.request.body)
+            except JSONDecodeError:
+                self._write_error(status_code=500, reason="Unexpected body, expected json.")
+
+    def post(self, runfolder):
```

Comment:
I think that we should add this functionality to the existing `GET` method. The reason for this is that we are not actually changing any state (which is what `POST` implies - more info on http verbs here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods). This means that we need to pick up the list of headers to ignore from the request parameters.

Reply:
Aha, I thought it was custom to use POST whenever you're sending data in the request, but the change of state distinction makes more sense to me. Will fix!